packages/web/src/js/utils/index.ts

```diff
@@ -7,7 +7,7 @@ const isElement = (object: any): boolean => {
   return typeof object.nodeType !== 'undefined';
 };
 
-const getElement = (object?: any): Node | Element | HTMLElement | null => {
+const getElement = (object?: any): SpiritElement => {
```

Comment:
Vidím, že to tak už bylo, ale nejde to otypovat lépe, než `any`?

Reply:
Well, make yourself a favor and try it. Maybe you will be better than me :-) Change type, run `yarn types`, 💥  Because both function `isElement` and `getElement` does some kind of type checking we can afford to use `any` in this case. But I am open to any! suggestion for better typing.

Reply:
Maybe some combination of `Element` and `HTMLElement` with `SpiritElement`?